src/index.ts

```diff
@@ -1,28 +1,33 @@
-import '/build/css/style.css'
-import { Movie_Search, Result } from './Movie'
-import { api_key } from './key.js'
+import { MovieSearch, Result } from './Movie'
+import { apiKey } from './key'
 
-const movieDisplay = document.getElementById('movie-display-container')!
+const movieDisplay = document.getElementById(
+  'movie-display-container'
+) as HTMLElement
 const movieSearchBox = document.getElementById(
   'movie-search-box'
 ) as HTMLInputElement
 
 // Get Search Term to call API
 function searchMovies(searchTerm: string): void {
   if (searchTerm.length > 0) {
+    // eslint-disable-next-line @typescript-eslint/no-floating-promises, @typescript-eslint/no-use-before-define
     loadMovies(searchTerm)
   }
 }
 // Fetch Data, Return JSON
-const fetchMovieList = async (searchTerm: string): Promise<Movie_Search> => {
-  const API_URL = `https://api.themoviedb.org/3/search/movie?api_key=${api_key}&query=${searchTerm}`
+const fetchMovieList = async (searchTerm: string): Promise<MovieSearch> => {
+  const API_URL = `https://api.themoviedb.org/3/search/movie?api_key=${apiKey}&query=${searchTerm}`
   const movieResp = await fetch(API_URL)
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-return
```

Comment:
I would do the same here as my previous comment. Again, just an example, it's not important for this project. If it feels time consuming, then just stick to what you have. 

The reason I gave the example is you can imagine you're working on a code base that has hundreds of files with thousands of lines, you wouldn't want to file an issue like "fix linting errors" because that might be like 50 different things and other developers on your team (or yourself in like a year when you get back to it) might not have enough time or context to figure them all out. And while the linter will help you find the location of the bugs, non-linting bugs might not be so obvious which lines of code are causing them, so it's a decent habit to track the TODO in the code itself as a short comment that links out to more details.

Reply:
it'll become more obvious in time

Reply:
Thanks for the scenario, I want to incorporate that habit early. I'm looking up a some methods to streamline it with VS Code. I'll add these to the next commit for sure.

Reply:
👍